### PR TITLE
Fix deletion commands

### LINKED ISSUE #7: Install fails on: "del ./build"
I tried to run the new version of the installer. The installer seems to work fine, but i can't run the project because "npm clean" is failing on "del ./build". the folder doesn't exist, but it still fails when i create.

i'm on Windows 10, running node 4.4.5. 

any suggestions would be helpful!

### PR DESCRIPTION
**Pull request by mfeltscher**
Use "del-cli" executable instead of "del" in order to not interfere with system commands on Windows - fixes #7

### CODE REVIEW

**Comment by buehler** [comment]
Have you tried it on a windows machine? :-)

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Fixed
+- Use `del-cli` executable instead of `del` in order to not interfere with system commands on Windows ([#7](/../../issues/7))
```
Comment (buehler) [inline CHANGELOG.md]: please use the full http link (when the package is somewhere installed, the link wouldn't work)

**Comment by mfeltscher** [comment]
@buehler No, I haven't. But there is a hint in [the del-cli documentation](https://github.com/sindresorhus/del-cli#usage) which clearly states `Since $ del is already a builtin command on Windows you need to use $ del-cli there.`

File: CHANGELOG.md (hunk 1 above)
Comment (mfeltscher) [reply CHANGELOG.md]: Adjusted in c20c01d - let's hope we never move that repository ;-)

**Pull request merged**